dockerfiles/agents/Dockerfile.agent

```diff
@@ -18,4 +18,4 @@ RUN chmod +x ./wait.sh
 
 USER $user
 
-CMD ./wait.sh ./startup.sh
```

Comment:
removed the wait - this is actually ngrok-wait.sh as it's renamed when copied onto the docker. No longer required as ngrok is disabled